source/features/clean-conversation-headers.tsx

```diff
@@ -60,6 +72,7 @@ async function init(signal: AbortSignal): Promise<void> {
 	observe([
 		'.gh-header-meta > .flex-auto', // Real
 		'.rgh-conversation-activity-filter', // Helper in case it runs first and breaks the `>` selector, because it wraps the .flex-auto element
+		'[class^="StateLabel"] + div > span',
```

Comment:
If this is so generic, we might want to use `:first-child` on `span` to avoid ever matching elements that we don't intend to match.